pkg/kubelet/prober/results/results_manager_test.go

```diff
@@ -18,11 +18,14 @@ package results
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
+const timeout = 30 * time.Second
```

Comment:
Brendon advocated to use `util.ForeverTestTimeout` for all test timeout like this.